src/parser/utils.ts

```diff
@@ -27,3 +27,24 @@ export function isMap(d: unknown): d is Map<unknown, unknown> {
 export function isDate(d: unknown): d is Date {
   return Object.prototype.toString.call(d) === '[object Date]';
 }
+
+/** @internal */
+export type StylizeFunction = (x: unknown, style: string) => string;
+export type InspectParameterFn = (x: unknown, options: unknown) => string;
+export function getBasicInspectParameterFn(): InspectParameterFn {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  return v => `${v}`;
+}
```

Comment:
This doesn’t feel like the most impactful question so please feel free to just ignore and resolve, but why the extra level of indirection? This could be

```suggestion
// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
export function basicInspectParameterFn: InspectParameterFn = v => `${v}`;
```

right?

Reply:
Yes, I forget functions are first class objects sometimes!